docs/modules/ROOT/pages/how-tos/custom-theme.adoc

```diff
@@ -40,27 +40,39 @@ parameters:
   keycloak:
     extraInitContainers:
       theme-provider:
-        image: company/keycloak-theme:v1.0.0
+        image: company/keycloak-themes:v1.0.0
         imagePullPolicy: IfNotPresent
         command:
           - sh
         args:
           - -c
           - |
             echo "Copying theme..."
-            cp -R /theme/* /company-theme
+            cp -R /themes/* /target/
         volumeMounts:
-          - name: theme
-            mountPath: /company-theme
+          - name: themes
+            mountPath: /target
+
     extraVolumes:
-      theme:
+      themes:
         emptyDir: {}
+
     extraVolumeMounts:
-      theme:
-        readOnly: true
+      theme-company:
+        name: themes
```

Comment:
I don't see any jsonnet changes in the postprocessing that makes use of the new parameter `name`.
Do you intend to do a separate PR for this?

Reply:
No, as it turns out everything was already fixed in my previous PR, making this one a documentation PR only :)

Reply:
Oh, and of course the tests that verify the intended behavior

Reply:
I was kinda skipping the tests :)
Please help me understand: Is the name parameter actually required now? That would make #79 a breaking change. Should we go for a non-breaking change if we default to the key name if the `name` parameter is missing?

Reply:
As far as I understood, the following will overwrite the `name` if it is present and otherwise use the key `k`. 

https://github.com/projectsyn/component-keycloak/blob/450ea1f79f9279d5d65b3c9144b40821325e1376/postprocess/extra-env.jsonnet#L13-L18

Reply:
Exactly, `name` wil be set to the key but overwritten if explicitly set. This was actually the case all along, but before #79 the mounts were deduplicated based on their `name` attribute (which is wrong since `name` identifies the volume, and not the volumeMount).

Reply:
thanks for clearing that up.